### DISCUSSION TITLE
Unable to Add Eufy Security Integration

### DISCUSSION THREAD

**Discussion by HufkieHome** [fuatakgun/eufy_security#170] [Q&A] [answered]
Hi

I am following the installation instructions and got stuck at point 6.3.5. After installing Eufy Security repository in HACS and restarting, I am not able to find the Eufy Security integration under configuration. 
I am running Hass on Pi4 and Core-2021.12.1
Where am I going wrong?

**Comment by fuatakgun** [comment, marked as answer]
this should be a cache problem, you should have resolved this already, right?

**Reply by HufkieHome** [reply]
Indeed. Just like magic it appeared. Thanks for the response... Love your work.
